src/stylesheets/root.scss

```
@@ -1,7 +1,6 @@
-@import 'transitions';
-@import 'scrollbar';
-@import 'disqus';
-@import url(../static/fonts/stylesheet.css);
+@use 'transitions';
+@use 'scrollbar';
+@import url(../public/fonts/stylesheet.css);
```

Stylelint (17.2.0)

[error] Expected quotes around "url" function argument (function-url-quotes)

(function-url-quotes)